packages/core/docs/commercetools/enterprise/use-wishlist.md

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
Please make this file with the same format as in here: https://github.com/DivanteLtd/vue-storefront/pull/5175/files#diff-f64570be73b90c75fb00ba8aa8f86d5792737047113e46d3621c5e5a74e66030

As you can see in there, common parts like general description, api, getters and basic examples are extracted to individual files and imported where needed.
This is done, because once we implement this features for other vendors, these parts will be shared.

It's important to keep headings (like `## Features`) in this file to properly create Table of Contents (`[[toc]]`).